grafana/dashboards/k6-prometheus.json

```diff
@@ -18,7 +18,9 @@
   "description": "Visualize k6 OSS results stored in Prometheus",
   "editable": true,
   "fiscalYearStartMonth": 0,
-  "graphTooltip": 0,
+  "gnetId": 19665,
+  "graphTooltip": 2,
+  "id": 29255,
```

Comment:
`id` is specific to the running Grafana instance. It should be removed from the JSON configuration.

Reply:
fixed!